chcemvediet/templates/main/homepage/login.html

```diff
@@ -36,6 +36,12 @@ <h2 id="{% trans 'main:homepage:login:anchor' %}">
       </div>
     </div>
   </div>
+  <div class="form-group">
+    <div class="col-sm-12">
+      <script src="https://www.google.com/recaptcha/api.js" async defer></script>
+        <div class="g-recaptcha" align="center" data-sitekey="{{ RECAPTCHA_PUBLIC_KEY|urlencode }}"></div>
```

Comment:
@viliambalaz Preco `urlencode`?

Reply:
:ok: Pouzivali sme to aj pri google search, tam sa vsak kluc dava priamo do url. ok, dakujem

Reply:
@viliambalaz Ano tam to bol argument do URL, tu to je normalna hodnota fieldu.